src/vue/backend/src/views/GuidedSetup.vue

```diff
@@ -18,70 +18,65 @@
                         <idx-list-item>Bullet list item 2</idx-list-item>
                         <idx-list-item>Bullet list item 3</idx-list-item>
                     </idx-list>
-                    <idx-button tag="a" href="#" size="lg">Let's Get Started</idx-button>
+                    <idx-button href="#" size="lg" @click="showDialog = true">Let's Get Started</idx-button>
                 </idx-block>
             </idx-block>
         </idx-container>
     </idx-fullscreen>
 </template>
 
-<style scoped lang="scss">
+<script>
+export default {
+    name: 'GuidedSetup'
+}
+</script>
+
+<style lang="scss">
     @import '~@idxbrokerllc/idxstrap/dist/styles/components/buttons';
+    @import '~@idxbrokerllc/idxstrap/dist/styles/components/dialog';
```

Comment:
You no longer need the dialog styles here since you are not importing the dialog component